xml/src/main/java/org/ehcache/xml/CacheManagerServiceConfigurationParser.java

```diff
@@ -34,4 +34,8 @@
   URI getNamespace();
 
   ServiceCreationConfiguration<T> parseServiceCreationConfiguration(Element fragment);
+
+  Class<T> getServiceCreationConfigurationType();
+
+  Element translateServiceConfiguration(ServiceCreationConfiguration<T> serviceCreationConfiguration);
```

Comment:
Document might be easier than element here, and is arguably more correct.  We know that the configs this thing is generating must honor an xsd so must be able to be validating document in their own right, not just fragments.

Reply:
You might want to confer with @chaktatha since I recommended switching to unparse in his PR.